src/man/sssd.conf.5.xml

```diff
@@ -2964,6 +2966,16 @@ min_id = 10000
 max_id = 20000
 enumerate = False
 </programlisting>
+	</para>
+	<para>
+            2. The following example shows configuration in IPA AD trust. AD having
```

Comment:
What about ... "configuration in AP AD trust, when AD has a child domain"?

Reply:
@amitkumar50 I'm sorry this review stalled.

I think @fidencio's suggestion was mostly about merging the first two sentences. And I do think that something like what you suggested would work. I would say "IPA AD trust where the AD forest consists of two domains in a parent-child structure". But feel free to phrase that first sentence differently if you disagree..